README.md

```diff
@@ -164,6 +164,11 @@ possible memory leak
 pointing the cursor and left or right clicking, as described in this
 [open issue](https://github.com/OpenAdaptAI/OpenAdapt/issues/145)
 
+#### Want to capture Chrome Browser Events as well ?
+
+- Follow the steps listed here for setup of a chrome (browser) extension: [Setup for Chrome Extension](https://github.com/OpenAdaptAI/OpenAdapt/pull/744#issuecomment-2212523232).
+
+
```

Comment:
This isn't really sufficient. We need a new section entitled "Capturing Browser Events" that contains the instructions in-line.

Reply:
Done here: ac65f13cc656e6b7c3d58735f8179d21a5094309